### PR TITLE
feat(api): add endpoint for roles

### PR DESCRIPTION
**Pull request by arminpatel**

### CODE REVIEW

File: api/api/roles/serializers.py [hunk 1]
```diff
@@ -0,0 +1,9 @@
+from rest_framework import serializers
+
+from .models import Roles
+
+
+class RolesSerializer(serializers.ModelSerializer):
```
Comment (Samy-33) [inline api/api/roles/serializers.py:6]: Let's keep it singular

Comment (Samy-33) [reply api/api/roles/serializers.py:6]: Also can you please rename `Roles` model to `Role`

File: api/api/roles/serializers.py [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+    class Meta:
+        model = Roles
+        fields = ['name', 'club', 'user', 'assigned_at', 'active']
```
Comment (Samy-33) [inline api/api/roles/serializers.py:9]: `assigned_at` should be auto populated?

File: api/api/roles/views.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
+from rest_framework.generics import CreateAPIView
+from rest_framework.permissions import IsAdminUser
+
+from .models import Roles
+from .serializers import RolesSerializer
+
+
+class CreateRolesView(CreateAPIView):
```
Comment (Samy-33) [inline api/api/roles/views.py:8]: CreateRoleView

**Review by Samy-33 — CHANGES_REQUESTED** [review]

File: api/api/roles/tests.py [hunk 4]
```diff
@@ -0,0 +1,80 @@
...
+                                  Role.ROLE_CORE_MEMBER,
+                                  Role.ROLE_FACULTY_INCHARGE,
+                                  Role.ROLE_CONVENER,
+                                  Role.ROLE_CO_CONVENER,
+                                  Role.ROLE_COUNSELLOR,
+                                  Role.ROLE_ASSOCIATE_COUNSELLOR])
+def testCreateRole_adminRequest_returnSuccesful(role):
+    # given
+    client = APIClient()
+    test_user = User.objects.create(email='test@user.com',
```
Comment (Samy-33) [inline api/api/roles/tests.py:52]: I don't understand how these test are succeeding without saving test_user?

Comment (arminpatel) [reply api/api/roles/tests.py:52]: > I don't understand how these test are succeeding without saving test_user?

No idea. My bet guess is due to `force_authenticate`.

Comment (Samy-33) [reply api/api/roles/tests.py:52]: Hmm, I'll check it out.,

**Review by Samy-33 — APPROVED** [review]

**Pull request merged**